include/nbl/asset/ICommandBuffer.h

```diff
@@ -287,6 +267,14 @@ class ICommandBuffer
     virtual void fillBuffer(buffer_t* dstBuffer, size_t dstOffset, size_t size, uint32_t data) = 0;
     virtual void updateBuffer(buffer_t* dstBuffer, size_t dstOffset, size_t dataSize, const void* pData) = 0;
 
+    virtual bool executeCommands(uint32_t count, cmdbuf_t** cmdbufs)
+    {
+        for (uint32_t i = 0u; i < count; ++i)
+            if (cmdbufs[i]->getLevel() != EL_SECONDARY)
+                return false;
+        return true;
+    }
```

Comment:
wait what?

Reply:
?

Reply:
why does ICommandBuffer have a virtual to execute commands?

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCmdExecuteCommands.html